jupyter_server/serverapp.py

```diff
@@ -546,6 +566,21 @@ class ServerApp(JupyterApp):
         password=(JupyterPasswordApp, JupyterPasswordApp.description.splitlines()[0]),
     )
 
+    default_services = [
+        'api',
+        'config', 
```

Comment:
Was the 'auth' handler left out on purpose?
```suggestion
        'auth',
        'config', 
```

Reply:
Oops, that wasn't on purpose!